operator/cmd/syncclusterconfig/sync.go

```diff
@@ -20,9 +20,14 @@ import (
 	"bufio"
 	"bytes"
 	"context"
+	"crypto/md5" // nolint:gosec// it's being used to hash a config
```

Comment:
Use sha1 or FNV instead please.

Reply:
I'd prefer FNV since this isn't really a crypto hash, it's faster, and we won't need to swap it out eventually for FIPS purposes if we ever go that route.